packages/image_picker/image_picker_ios/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -133,7 +133,12 @@ - (void)pickImageWithPHPicker:(int)maxImagesAllowed API_AVAILABLE(ios(14)) {
 
   self.maxImagesAllowed = maxImagesAllowed;
 
-  [self checkPhotoAuthorizationForAccessLevel];
+  BOOL usePHAsset = [[_arguments objectForKey:@"requestFullMetadata"] boolValue];
```

Comment:
Should the BOOL name to be `requestFullMetadata`? Or explain why `usePHAsset` is the same as `requestFullMetadata` in a comment